spec/support/page_objects/teacher_interface/documents/check_uploaded_files.rb

```diff
@@ -0,0 +1,19 @@
+module PageObjects
+  module TeacherInterface
+    class CheckUploadedFilesCard < SitePrism::Section
```

Comment:
There's a pretty much identical one of these in each page object. Should we make a generic `Card` section? Or maybe just define them inline in the PageObejct if it's only going to have an `h2`?

Reply:
It might actually not be necessary any longer. I can't fully remember what I actually put them in to achieve. I think they can just be defined in the actual pageobject itself where needed